pkg/operator/controller/ingress/load_balancer_service_test.go

```diff
@@ -779,6 +820,9 @@ func Test_desiredLoadBalancerService(t *testing.T) {
 				TargetPort: intstr.FromString("https"),
 			}}, svc.Spec.Ports)
 			assert.Equal(t, "None", string(svc.Spec.SessionAffinity))
+			if tc.expectedFloatingIP != "" {
+				assert.Equal(t, tc.expectedFloatingIP, svc.Spec.LoadBalancerIP)
+			}
```

Comment:
No need to filter out for empty floating IP, that removes coverage for ensuring it doesn't get added for internal load balancers.

```suggestion
assert.Equal(t, tc.expectedFloatingIP, svc.Spec.LoadBalancerIP)
```